index/get_manifest_info.py

```diff
@@ -76,26 +80,40 @@ def _get_library(manifest_info):
     return library
 
 
+def _get_library_collection_code(manifest_info):
+    """ Get Primo lsr01 Collection Code given repository """
```

Comment:
Could be rewritten to the below code(untested).  At least change to use if/elif/else instead of just if

lcc = {
  'UNDA': 'UNDA ARCHV',
  'SPEC': 'SPEC',
  'SNITE': 'SNITE'
}
return lcc.get(manifest_info['repository'].upper(), '')

Reply:
Great suggestion.  I have made these changes.